### ISSUE TITLE
BUG] ERROR: relation "oc_appconfig" does not exist at character 57 with new install of latest version ONLY

### ISSUE THREAD

**Issue by alamoudimoh** [nextcloud/docker#2284]
the below error only happens when installing a fresh installtion of 29.0.5-ls335

i used version 29.04 and it works fine,
i used version 29.04 and upgraded to latest tag and it worked fine, only with new installation of latest

```
Internal Server Error

The server encountered an internal error and was unable to complete your request.
Please contact the server administrator if this error reappears multiple times, please include the technical details below in your report.
More details can be found in the server log.
```

Steps To Reproduce
create new docker with PostgreSQL and nextcloud using nextcloud latest tag
installtion complete successfully
start the intial setup.
once the database start to connect you get the following error

**Comment by joshtrichards** [comment]
`29.0.5-ls335` sounds like a tag used for the LSIO image, not not our image.

Though the underlying cause may be similar to a bug here: #2283

**Comment by alamoudimoh** [comment]
> `29.0.5-ls335` sounds like a tag used for the LSIO image, not our image.
> 
> Though the underlying cause may be similar to a bug we're currently tracking here: #2283

Thanks Josh,

Yes it is Liso image qnd i open an issue with them and they refer me here, so I install the latest image here and it has the same issue

**Comment by joshtrichards** [comment]
Well, upstream of that image would be Nextcloud `Server` not this image here. But no worries. :-)

Fixed upstream in nextcloud/server#47521

Closing this as a duplicate of #2283 since there's nothing uniquely actionable being tracked in this issue now.
